src/wp-content/themes/twentynineteen/sass/blocks/_blocks.scss

```
@@ -1046,3 +1046,16 @@
 		color: #FFF;
 	}
 }
+
+.wp-block-navigation-item {
+
+	.wp-block-navigation-submenu__toggle {
+		background: transparent;
+		outline-offset: 0;
+		border-radius: 0;
+	}
+
+	&.open-on-click .wp-block-navigation-submenu__toggle {
+		padding: 0 0.85em 0 0;
```

```suggestion
		padding-right: 0.85em;
```
Could we go with `padding-right`? other side is default `0`.

---

This uses the [right padding in the block's styles](https://github.com/WordPress/gutenberg/blob/2379ffd99aca2ba0a38c756bbb343efedf9b0e55/packages/block-library/src/navigation/style.scss#L317) of `0.85em`, which was in WordPress 6.4. The plugin now has zero left padding to override _browser_ styles, though I wanted to remove the _theme's_ padding on the top and bottom too.

Maybe this patch should only set the top and bottom (and/or left) to zero in case the right value ever changes in the block styles.